src/library/searchquery.cpp

```diff
@@ -168,10 +168,13 @@ QString NotNode::toSql() const {
 
 TextFilterNode::TextFilterNode(const QSqlDatabase& database,
         const QStringList& sqlColumns,
-        const QString& argument)
+        const QString& argument,
+        const StringMatch matchMode)
         : m_database(database),
           m_sqlColumns(sqlColumns),
-          m_argument(argument) {
+          m_argument(argument),
+          m_matchMode(matchMode) {
+    qRegisterMetaType<StringMatch>("StringMatch");
```

Comment:
`StringMatch` doesn't need to be a metatype afai can tell.
```suggestion
```
same unnecessary registration in `SearchQueryParser` ctor.

Reply:
Yes, I can build just fine without registration on Linux, but I remember this was(?) necessary for some OS/compiler when I added `enum class FocusWidget` (CI failed to built without it).

Reply:
well, in this case, I'm pretty sure it shouldn't

Reply:
I'll push a test commit. If it's not needed (anymore?), great.